models/asset.js

```diff
@@ -4,11 +4,21 @@ var Schema = mongoose.Schema;
 var assetSchema = Schema({
 	name:String,
 
-	content:String,
+	category:String,
 
-	createDate:Date,
+	version:{
+		no:Number,
+		date:Date
```

Comment:
yes, version should be a simple string as I mentioned. Don't make simple complex.